jwst/pipeline/calwebb_tso3.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import unicode_literals, absolute_import
+
+import os
+
+from astropy.table import vstack
+
+from ..stpipe import Pipeline
+from .. import datamodels
+
+from ..outlier_detection import outlier_detection_tso_step
+from ..outlier_detection import outlier_detection_scaled_step
+from ..tso_photometry import tso_photometry_step
+from ..extract_1d import extract_1d_step
+from ..white_light import white_light_step
+
+__version__ = "0.7.8"
+
+
+class TSO3Pipeline(Pipeline):
```

Comment:
OK, so because:
1) foolish consistency is the hobgoblin of little minds, and
2) I have a little mind,
we will therefore use all caps for acronyms that appear in class names.